### PR TITLE
Last minute MacOS build fixes

### PR DESCRIPTION
**Pull request by jtanx**
If all goes well, I'll be using the travis mac build output for the release. The main things are:

* Disable listening to mac events, this is broken, and causes weird interaction with FontForge (e.g. doesn't show any windows on start, and only shows something when you click on the dock, it also doesn't quit when all the windows are closed). 
* Pin Pango to 1.42.2, to work around #3343

### CODE REVIEW

**Comment by jtanx** [comment]
So this does work, the pango pinning was successful. Just playing around with re-enabling listening to events, to see if I an get it to work with file open events

**Comment by skef** [comment]
I'm not sure we should have any dependencies on a project that has not only a *bug*, but a bug that causes a *crash*.  People might get confused and think our project's code could be responsible for that sort of thing.

**Comment by jtanx** [comment]
Pango is a pretty core requirement to FontForge, but it's also hard to get anyone to fix things on the quartz backend. fwiw, I did try to debug it, but I was struggling to even get a version of pango with debugging symbols working with everything else to see what the problem was.

**Comment by MaddTheSane** [comment]
> it also doesn't quit when all the windows are closed[...]

That is standard behavior for Mac apps. But because it doesn't use the Mac menu bar, it does make using the app without a window impossible.

File: travis-scripts/_FontForge [hunk 1]
```diff
@@ -132,7 +132,7 @@ export PATH="$PATH:$bundle_bin"
 echo $WRAPPER $bundle_bin/fontforge > /tmp/oo
 
 if [ $# -eq 0 ] ; then
-    ( exec $WRAPPER $bundle_bin/fontforge -new & )
+    ( exec $WRAPPER $bundle_bin/fontforge -new  )
```
Comment (jtanx) [inline travis-scripts/_FontForge]: This was so annoying, with this it actually uses the FontForge icon now, and it semi-works with opening files from Finder.

**Comment by jtanx** [comment]
The reason things are semi broken is that by the time FF installs the carbon (!!!!) event handlers, it's a bit too late, it's already received the open event (and if it wasn't yet started, the open file event). I believe this is because GDK pumps the quartz(?)/cocoa(?) event loop (couple with us calling GDrawProcessPendingEvents a few times in the super complicated startup sequence), whereas before, it wouldn't have been pumped until the event handlers were installed. 

Anyway, I'm leaving this as is, it's good enough for now.

File: travis-scripts/_FontForge (hunk 1 above)
Comment (skef) [reply travis-scripts/_FontForge]: Kinda makes sense. What would it mean to background a shell `exec` anyway ?

Comment (jtanx) [reply travis-scripts/_FontForge]: It was probably a workaround for some other issue back when it was an X application, I vaguely recall mention of the dock icon continuously bouncing (this doesn't happen any more from what I can tell).

**Pull request merged**